plugins/spamassassin.js

```diff
@@ -105,7 +105,7 @@ exports.hook_data_post = function (next, connection) {
         if (!connection.transaction) return next();
 
         if (spamd_response.headers && spamd_response.headers.Tests) {
-            spamd_response.tests = spamd_response.headers.Tests;
+            spamd_response.tests = spamd_response.headers.Tests.replace(/\s/g, '');
```

Comment:
```suggestion
            spamd_response.tests = spamd_response.headers.Tests.replace(/\s+/g, ' ');
```
There might be a good reason this isn't a good fix, but the one included here looks odd to me.

Reply:
I'm not looking to replace multiples of WS (your suggestion) with a single space (your suggestion). There's currently some SMTP line folding included in the string. My goal is to strip out _all_ the WS chars (in this case, sets of `\r\n\t` and/or `\r\n\s+`).